### PR TITLE
Extract the OpenSlides 3 client

### PR DESCRIPTION
**Pull request by tsiegleauq**
Extracts the client for OpenSlides 4

Add a docker config for both
production build and developing toolchain

Set and adjust the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,164 @@
-# OpenSlides client
+# OpenSlides Client
 
-Webclient for OpenSlides which ...
+Web client for OpenSlides which is written in Angular.
```
Comment (normanjaeckel) [inline README.md]: Hm. The ... which ... clause was ment to describe what this part of OpenSlides is doing. But if "client" is self explaining you may write only "Web client for OpenSlides 4+". Please also adjust repository description in GitHub.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,164 @@
-# OpenSlides client
+# OpenSlides Client
```
Comment (normanjaeckel) [inline README.md]: OpenSlides client or OpenSlides Client? @emanuelschuetze

**Comment by normanjaeckel** [comment]
Maybe also add ``.travis.yml``. After that we can enable status checks for further pull requests.

File: README.md (hunk 2 above)
Comment (emanuelschuetze) [reply README.md]: Let us use always capital letters for service names! As in our [architecture diagram](https://raw.githubusercontent.com/OpenSlides/OpenSlides/openslides4-dev/docs/OpenSlides4-systemarchitecture.png). E.g. "OpenSlides Backend Service", not "OpenSlides backend service".

**Comment by tsiegleauq** [comment]
Added `.travis.yml`
Adjusted the readme even more
Extended the "which ..." clause

File: README.md (hunk 2 above)
Comment (normanjaeckel) [reply README.md]: Ok.

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**